pkgs/development/python-modules/git-sim/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  installShellFiles,
+  gitpython,
+  manim,
+  opencv4,
+  typer,
+  pydantic,
+  fonttools,
+  git-dummy,
+}:
+
+buildPythonPackage rec {
+  pname = "git-sim";
+  version = "0.3.5";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "initialcommit-com";
+    repo = "git-sim";
+    rev = "v${version}";
+    hash = "sha256-4jHkAlF2SAzHjBi8pmAJ0TKkcLxw+6EdGsXnHZUMILw=";
+  };
+
+  postPatch = ''
+    sed -i '/opencv-python-headless/d' pyproject.toml
+  '';
+
+  propagatedBuildInputs = [
+    gitpython
+    manim
+    opencv4
+    typer
+    pydantic
+    fonttools
+    git-dummy
+  ];
+
+  # https://github.com/NixOS/nixpkgs/commit/8033561015355dd3c3cf419d81ead31e534d2138
+  makeWrapperArgs = [ "--prefix PYTHONWARNINGS , ignore:::pydub.utils:" ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    for shell in bash fish zsh; do
+      $out/bin/git-sim --show-completion $shell > git-sim.$shell
+      installShellCompletion git-sim.$shell
+    done
+
+    ln -s ${git-dummy}/bin/git-dummy $out/bin/
```

Comment:
Why not put that into the wrapper under the PATH env?

Reply:
I am not sure what you mean in this case, would `git-dummy` still be available in the user's `PATH`? `git-sim` itself doesn't need `git-dummy` in its `PATH`, it only uses it for tests, and even then it uses it as a module which is covered by `dependencies`.

Reply:
In that case, why include git-dummy in the output for git-sim?

Reply:
It was to honour upstream's expectation that installing `git-sim` includes `git-dummy`